packages/hooks/src/useConfig.ts

```diff
@@ -1,8 +1,28 @@
-import { useContext } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { LiteflowContext } from './context'
 import { Config } from './graphql'
 
-export default function useConfig(): Promise<Config> {
-  const { config } = useContext(LiteflowContext)
-  return config
+// This type can be reused for any other read-only hook
+type Result<T = any> = {
+  data?: T
+  error?: Error
+}
+
+export default function useConfig(): Result<Config> {
+  const { sdk } = useContext(LiteflowContext)
+  const [config, setConfig] = useState<Config>()
+  const [error, setError] = useState<Error>()
+
+  useEffect(() => {
+    sdk
+      .GetConfig()
+      .then(({ config }) => config)
```

Comment:
Missing the setConfig:
```suggestion
      .then(({ config }) => setConfig(config))
```

Reply:
fixed in [49187fb](https://github.com/liteflow-labs/liteflow-js/pull/120/commits/49187fb5debf5130f059c745d1c55ebdb0257151)